### PR TITLE
MESH-1622/ create script to distribute POLYX

### PR DESCRIPTION
**Pull request by JMoore96**
How to run script:
`cd scripts/cli/util`
`node <file_name>.js --account <private_key or mnemonic phrase> --amount <amount>`

If you are using mnemonic phrase as the account wrap it in quotes.
### modified logic

- added distributePOLYX.js

### CODE REVIEW

File: scripts/cli/util/distributePOLYX.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+const args = minimist(process.argv.slice(2), {
+    string: 'account'
+});
+
+async function main() {
+    const api = await reqImports.createApi();  
+    const account = await reqImports.generateEntityFromUri(api, args.account);  
+    const amount = args.amount; 
+
+    let empty_did = "0x0000000000000000000000000000000000000000000000000000000000000000"
```
Comment (CJP10) [inline scripts/cli/util/distributePOLYX.js:16]: const?

**Review by CJP10 — APPROVED** [review]

File: scripts/cli/util/distributePOLYX.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+async function batchAtomic(api, sender, receivers, amount) {
+
+    let tx;
+    let txArray = [];
+    let batch;
+    let batchTx;
+    let senderDid = await api.query.identity.keyToIdentityIds(sender.publicKey);
+    let batchSize = 10;
```
Comment (Centril) [inline scripts/cli/util/distributePOLYX.js:39]: Consider not upfront declaring stuff here...

File: scripts/cli/util/distributePOLYX.js (hunk 1 above)
Comment (JMoore96) [reply scripts/cli/util/distributePOLYX.js]: thanks for the advise, changed empty_did to a const.

File: scripts/cli/util/distributePOLYX.js (hunk 2 above)
Comment (JMoore96) [reply scripts/cli/util/distributePOLYX.js]: understood, made the changes.

**Comment by adamdossa** [comment]
@JMoore96 Please can you run on PME using //Alice - you can airdrop 1 POLYX to each primary key.

**Review by adamdossa — APPROVED** [review]

**Pull request merged**